lib/screens/alerts/alert.ex

```diff
@@ -1,6 +1,7 @@
 defmodule Screens.Alerts.Alert do
   @moduledoc false
 
+  alias Screens.Alerts.Alert
```

Comment:
This is the module we're in, so conventionally we'd refer to it using `__MODULE__` rather than aliasing it. However it looks like this was only added to allow putting `Alert.t()` in the typespec for the new function, which can also be expressed as just `t()`.